hbase/src/main/scala/geotrellis/spark/io/hbase/HBaseRDDWriter.scala

```diff
@@ -51,25 +68,59 @@ object HBaseRDDWriter {
       }
     }
 
+    val _recordCodec = KeyValueRecordCodec[K, V]
+    val kwWriterSchema = KryoWrapper(writerSchema) // Avro Schema is not Serializable
+
     // Call groupBy with numPartitions; if called without that argument or a partitioner,
     // groupBy will reuse the partitioner on the parent RDD if it is set, which could be typed
     // on a key type that may no longer by valid for the key type of the resulting RDD.
-
     raster.groupBy({ row => decomposeKey(row._1) }, numPartitions = raster.partitions.length)
       .foreachPartition { partition: Iterator[(Long, Iterable[(K, V)])] =>
         if(partition.nonEmpty) {
           instance.withConnectionDo { connection =>
             val mutator = connection.getBufferedMutator(table)
+            val _table = instance.getConnection.getTable(table)
```

Comment:
Ah, sry I missed it %) we already have a `connection`

Should be:
```scala
val _table = connection.getTable(table)
```

`withConnectionDo` is a magic function which closes it after the `block` of code is done: 

```
def withConnectionDo[T](block: Connection => T): T = {
    val connection = getConnection
    try block(connection) finally connection.close()
  }
```